source/includes/options-mongosqld.yaml

```diff
@@ -671,4 +671,37 @@ args: <service-name>
 description: |
   Description of the system service which runs :binary:`~bin.mongosqld`.
 optional: true
+---
+program: mongosqld
+name: gssapiHostname
+directive: option
+args: <hostname>
+description: |
+  .. versionadded:: 2.5
+
+  Hostname that serves Kerberos.
+optional: true
+default: First IP address for :setting:`net.bindIp`.
+---
+program: mongosqld
+name: gssapiServiceName
+directive: option
+args: <service-name>
+description: |
+  .. versionadded:: 2.5
+
+  Name of the system service which runs the Kerberized :binary:`~bin.mongosqld`.
```

Comment:
this option doesn't really have anything to do with a system service -- it is the service name component of the kerberos SPN. We can use the mongod docs for security.sasl.serviceName as a template for what this should look like